src/models/user/influencers.rs

```diff
@@ -0,0 +1,170 @@
+use crate::db::graph::exec::retrieve_from_graph;
+use crate::db::kv::index::sorted_sets::SortOrder;
+use crate::types::DynError;
+use crate::types::StreamReach;
+use crate::types::Timeframe;
+use chrono::Utc;
+use serde::{Deserialize, Serialize};
+use std::ops::Deref;
+use utoipa::ToSchema;
+
+use crate::queries;
+use crate::RedisOps;
+
+const GLOBAL_INFLUENCERS_PREFIX: &str = "Cached:Influencers";
+
+#[derive(Deserialize, Serialize, ToSchema, Debug, Clone)]
+pub struct Influencer {
+    pub id: String,
+    pub score: f64,
+}
+
+// Define a newtype wrapper
+#[derive(Serialize, Deserialize, Debug, ToSchema, Default, Clone)]
+pub struct Influencers(pub Vec<Influencer>);
+
+impl RedisOps for Influencers {}
+
+// Create a Influencers instance directly from an iterator of Influencer items
+// Need it in collect()
+impl FromIterator<Influencer> for Influencers {
+    fn from_iter<I: IntoIterator<Item = Influencer>>(iter: I) -> Self {
+        Influencers(iter.into_iter().collect())
+    }
+}
+
+// Implement Deref so Influencers can be used like Vec<String>
+impl Deref for Influencers {
+    type Target = Vec<Influencer>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
```

Comment:
The use of _Influencer_ struct feels unnecessary, as a tuple `(score, pubky)` seems to capture the same logic effectively. A vector of such tuples, `Vec<(score, pubky)>`, _Influencers_ struct now, would suffice, reducing complexity without losing clarity

Reply:
I wouldn't call it unnecessary.  
Having a struct for it means in the future we can easily add more properties to it if needed, without having to refactor a lot of code.  
This isn't hurting readability, performance or consistency, so I don't think it hurts to keep it here.  
What do you think?

Reply:
You know what I think from the beginning 😁. That data about users will always go to get `UserViews` so now and in the future, I do not think that we will need that struct. @SHAcollision any opinion about that?